examples/basic/main.tf

```diff
@@ -29,22 +30,6 @@ resource "azurerm_public_ip" "example" {
   sku                 = "Standard"
 }
```

Comment:
Same here. I think at some point the firewall was removed due to cost reasons but these resources were forgotten :/